modules/apps/frontend-js/frontend-js-a11y-web/src/main/resources/META-INF/resources/components/Violations.tsx

```diff
@@ -212,11 +201,7 @@ export default function Violations({
 									id
 								)}
 								key={id}
-								onClick={() => {
-									if (next) {
-										next({violationIndex: index});
-									}
-								}}
+								onClick={() => next!({violationIndex: index})}
```

Comment:
Is this `!` a typo?

Reply:
This is Non-null assertion operator, we can tell the TypeScript compiler that this method is not null or optional for example. https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

In this case, I think you had pasted the optional `next` method because we passed it via `cloneElement` now through `StackNavigator` and the TypeScript doesn't handle that well because it would force passing this method when adding the component as children of ` StackNavigator`.